meteor-app/imports/ui/Login/login.tsx

```diff
@@ -22,13 +22,24 @@ export const Login = () => {
   }
 
 	const handleSubmit = () => {
-		setUsername('')
+		setUsername('') 
 		setPassword('')
 		Meteor.loginWithPassword(username, password, (error) => {
 			if(error) console.log(error);
 			else {
-				setUserId(Meteor.userId())
-				history.push('report-list')
+        var role = ""
+        Meteor.call('Get_User_Role', Meteor.userId(), (error : any, result : any) => {
+          if (error) {
+            console.log(error)
+          } else {
+            role = result; 
```

Comment:
you could instead, rename `result` within the callback to `role`

this prevents the need to make a reference with `role = result`